File: Baekjoon/15552.py
# import sys
# input = sys.stdin.readline

T = int(input())

A= []
B= []

for _ in range(T):
    a, b =map(int, input().split())
    A.append(a)
    B.append(b)

for i in range(T):
    total = A[i]+B[i]
    print(total)

File: JavaScript/Baekjoon/15552.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = fs.readFileSync(filePath).toString().trim().split("\n");

let answer = "";

for (let i = 1; i <= input[0]; i++) {
  tmp = input[i].split(" ");
  answer += parseInt(tmp[0]) + parseInt(tmp[1]) + "\n";
}

console.log(answer);
